contents/handbook/small-teams/experimentation/objectives.mdx

```diff
@@ -1,35 +1,48 @@
-    - bulk add emails to cohorts
-    - view cohorts by organization
-    - edit static cohorts
-    - cohort creation improvement, flexible user additions
-    - cohorts by groups
-- **Key Result:** Release beta of developer integrated features with 5 users
-    - connect codebase and show feature flag coverage by branch
-    - reorient feature flag page to show flags that are active AND implemented
-    - Opportunity: Allow debugging integrations to be added and do issue tracking related to possible active feature flags
-    - (stretch goal) CI implementation that allows people to check flags in code deployment against what's active + exists
+### Objective: Increase deployability of Feature Flags
 
 * **Rationale:**
-    * We want feature flags to be used across larger engineering teams. Currently we aren't solving their needs fully but we have a clear path to doing so.
-
-### Objective: Launch Direct SQL Querying Beta
+     * Feature flags is a high friction feature. Unlike adding a snippet or simple line to track an event, engineers and product owners need to plan for their product to work with the flag on and off. 
+     To fully enable engineers to build with feature flags, we should make it remarkably easy to create flags.
+ 
+- **Key Result:** Activation rate of users using flags by 20% (Base metric TBD)
+- **Key Result:** TBD
+    
+Possible avenues:
+- Auto detection of feature flags being sent that don't have a corresponding flag in posthog and easy creation (like github asking for branch to PR)
+- API Enablement. Create experiments or feature flags using posthog API.
+- Better hook and time to deployment with feature flag creation when someone lands on the feature flag page
+- Less related but important: JSON flags, feature flag resiliency
 
-* **Key Result:**
-     * Five beta users who retain for three weeks
-* **Key Result:**
-     * Understand the possible product directions to extend direct sql querying
+### Objective: Understand similarities between users throughout posthog
 
 * **Rationale:**
-     * To fully support our technical users, we need to allow them to ask any sort of question about their data that might not be fit for an interface
+     * Help engineers stay focused on building. Give them as close to the answers and insights as possible with ample proof and dropoff points for further exploration.
  
+- **Key Result:** Internal discoveries around different segments of users and groups
+- **Key Result:** 
+
+ Possible avenues:
+- Cohort correlation. What are property and behavioral similarities between users in this cohort
+- Experiment correlation. What are property and behavioral similarities between successful and failing variants
+
+
+### Objective: Nail feature analysis
+
+* **Rationale:**
+     * We have internal power users on the growth team who actively find value in deploying with experiments. Making the act of quick insights for features seamless sets us up to deliver an opinionated workflow
+
```

Comment:
Because you asked for metrics Eric: I don't think this objective is directly measurable, but long-term, improvements like this should increase experiment retention